apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2147,7 +2162,8 @@ const saveConversationEpic: AppEpic = (action$) =>
     filter(
       (action) =>
         ConversationsActions.saveConversation.match(action) &&
-        !action.payload.isMessageStreaming, // shouldn't save during streaming
+        !action.payload.isMessageStreaming &&
+        !action.payload.isPlayback, // shouldn't save during streaming or if Playback
```

Comment:
I don't think it's good idea to not save any Playback changes.
Need to double check that Playback-conversation is saved after creation

Reply:
Converted this changes.

Now works as expected.